apps/point-of-sale/src/stores/cart.store.ts

```diff
@@ -57,7 +57,7 @@ export const useCartStore = defineStore('cart', {
     async setBuyer(buyer: BaseUserResponse | null) {
       this.buyer = buyer;
       if (buyer) {
-        const response = await apiService.balance.getBalanceId(buyer.id);
+        const response = await apiService.balance.getBalanceId(buyer.id).catch(this.buyerBalance = null);
```

Comment:
If somewhere in this process an error occurs (like when a user to not have permission to get another users balance), the value is set to null, so no balance is rendered.